src/generate/CodeModelAzImpl.ts

```diff
@@ -2269,10 +2360,27 @@ export class CodeModelCliImpl implements CodeModelAz {
         return extensions && 'x-ms-examples' in extensions ? extensions['x-ms-examples'] : {};
     }
 
+    public get Example(): unknown {
+        return this.Examples[this.currentExampleIndex];
```

Comment:
why it returns unknown?

Reply:
Because this will return the original swagger example, it's actually an object, but this tslint hint that we shouldn't use object type as it barely no better than unknown.